compass/ocean/tests/global_ocean/mesh/arm10to60/arm10to60.cfg

```diff
@@ -0,0 +1,57 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = tanh_dz
+
+# Number of vertical levels
+vert_levels = 64
+
+# Depth of the bottom of the ocean
+bottom_depth = 6000.0
+
+# The minimum layer thickness
+min_layer_thickness = 2.0
+
+# The maximum layer thickness
+max_layer_thickness = 210.0
+
+
+# options for global ocean testcases
+[global_ocean]
+
+## config options related to the initial_state step
+# number of cores to use
+init_cores = 36
+# minimum of cores, below which the step fails
+init_min_cores = 8
+# maximum memory usage allowed (in MB)
+init_max_memory = 1000
+
+## config options related to the forward steps
+# number of cores to use
+forward_cores = 720
+# minimum of cores, below which the step fails
+forward_min_cores = 144
+# maximum memory usage allowed (in MB)
+forward_max_memory = 1000
+
+## metadata related to the mesh
+# the prefix (e.g. QU, EC, WC, SO)
+prefix = ARM
+# a description of the mesh and initial condition
+mesh_description = MPAS Arctic Regional Mesh (ARM) for E3SM version
+                   ${e3sm_version}, with ${min_res}-km resolution in the Arctic
+                    and ${levels} vertical levels
```

Comment:
@darincomeau and @milenaveneziani, please suggest updated text for this description.  It goes into the metadata for every NetCDF file produced for this mesh in compass.

Reply:
We have been asked to move away from the `ARM` acronym because it is confused with the ARM observational program. Maybe we could use ARRM as before (Arctic Regional Refined Mesh).